### PR TITLE
linting set up

### PR DESCRIPTION
**Pull request by ayusharora99**
- so that code is consistent

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          key: v1-dependencies-{{ checksum "package.json" }}
+
+      # run tests!
+      - run: npm test
+
+workflows:
+  version: 2
+  main:
+    jobs:
+      - test
```
Comment (nahtnam) [inline .circleci/config.yml]: add a new line at the end of the file

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "airbnb-base"
```
Comment (nahtnam) [inline .eslintrc.js]: 2 spaces instead of 4

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+stages:
+  - test
+
+test:
+  stage: test
+  script:
+    - npm install
+    - npm test
+  tags:
+      - docker
```
Comment (nahtnam) [inline .gitlab-ci.yml]: delete this file

File: package.json [hunk 4]
```diff
@@ -10,16 +10,16 @@
...
     "build": "rescripts build",
-    "test": "rescripts test",
     "eject": "react-scripts eject",
     "dev": "concurrently \"BROWSER=none npm start\" \"wait-on http://localhost:3000 && electron .\""
   },
   "rescripts": [
     "env"
   ],
-  "homepage": "./",
+  "homepage": "http://./",
```
Comment (nahtnam) [inline package.json]: why did you change this?

File: package.json [hunk 5]
```diff
@@ -10,16 +10,16 @@
     "react-scripts": "3.2.0"
   },
   "scripts": {
+    "test" : "eslint *.js src/*",
     "start": "rescripts start",
     "build": "rescripts build",
-    "test": "rescripts test",
```
Comment (nahtnam) [inline package.json:15]: My bad, i think its best you combine both of these commands: `eslint *.js src/* && rescripts test`

File: package.json [hunk 6]
```diff
@@ -41,6 +41,20 @@
     "concurrently": "^5.0.0",
     "electron": "^6.0.12",
     "electron-builder": "^21.2.0",
+    "eslint": "^6.5.1",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.18.2",
     "wait-on": "^3.3.0"
+  },
+  "description": "## Prerequisites",
```
Comment (nahtnam) [inline package.json]: remove this

**Review by nahtnam — CHANGES_REQUESTED** [review]
fix

File: .circleci/config.yml (hunk 1 above)
Comment (ayusharora99) [reply .circleci/config.yml]: adding

File: .eslintrc.js (hunk 2 above)
Comment (ayusharora99) [reply .eslintrc.js]: spacing

File: .gitlab-ci.yml (hunk 3 above)
Comment (ayusharora99) [reply .gitlab-ci.yml]: deleting

File: package.json (hunk 4 above)
Comment (ayusharora99) [reply package.json]: no clue how that changed, changing back to former

File: package.json (hunk 5 above)
Comment (ayusharora99) [reply package.json:15]: combining

File: package.json (hunk 6 above)
Comment (ayusharora99) [reply package.json]: removing

**Review by nahtnam — APPROVED** [review]
